### ISSUE TITLE
The speed of the same model is rather slower than MXNet

### ISSUE THREAD

**Issue by Sun1992** [hpi-xnor/BMXNet#39]
Hi,

Thanks for such good work. I used the same structure on both MXNet and BMXNet. However, the speed of training process on MXNet is 400 imgs/sec meanwhile the speed on BMXNet is 30 imgs/sec. I checked the GPU and it worked. I use TitanXP and CUDA 8. Do you know how to solve this issue?

**Comment by yanghaojin** [comment]
Hello,
from your post, I cannot find the problem directly. Further information is required, on which kind of architecture your test is running (CNN, RNN)? Have you checked the MXNet's version under BMXNet , and the version in your MXNet? If the version of BMXNet is behind the MXNet, the problem might be solved by next upgrade. Pls make sure CuDNN works for both framework equally, and pls make sure you built both projects in the same building mode "Release or debug" etc. 
Basically, if you use BMXNet specific layers such as "q_cudnn_convolution*", it will be slower than the original Conv layer (but obviously, this is not your case). Otherwise, there should be no difference...

**Comment by Sun1992** [comment]
Hi haojin,
Thanks for your reply. I changed the cmake mode to release and the speed was as fast as mxnet. Many thanks.
